test/Libraries/RevitIntegrationTests/LevelTests.cs

```diff
@@ -10,35 +11,37 @@ namespace RevitSystemTests
     [TestFixture]
     class LevelTests : SystemTest
     {
-        [Test]
+        [Test, Category("Failure")]
```

Comment:
There is no point adding a Failing test, unless we have a defect for this. It would be good to put a link of defect on this test.